### ISSUE TITLE
Export to different worksheet with cryptic names fail

### ISSUE THREAD

**Issue by ArneD** [UweKeim/ZetaResourceEditor#18]
Hi,

We encountered the following error when exporting each file group to different sheet and using the option 'use cryptic names'.  
Seems the GUID generation is not correct.

Edit: After closing and re-opening the program, it works.

```System.Data.DuplicateNameException

-----------------

A DataTable named '00000000-0000-0000-0000-0000...' already belongs to this DataSet.

-----------------

at System.Data.DataTableCollection.RegisterName(String name, String tbNamespace)
   at System.Data.DataTable.set_TableName(String value)
   at ZetaResourceEditor.RuntimeBusinessLogic.ExportImportExcel.Export.ExcelExportController.doProcess(PreparedInformation preparedInformation, BackgroundWorker bw, Int32 fileGroupsLength, ICollection`1 savedFiles) in c:\P\ZetaResourceEditor\Source\RuntimeBusinessLogic\ExportImportExcel\Export\ExcelExportController.cs:line 451
   at ZetaResourceEditor.RuntimeBusinessLogic.ExportImportExcel.Export.ExcelExportController.Process(BackgroundWorker bw) in c:\P\ZetaResourceEditor\Source\RuntimeBusinessLogic\ExportImportExcel\Export\ExcelExportController.cs:line 304
   at ZetaResourceEditor.UI.ExportImportExcel.ExcelExportWizardForm.progressBackgroundWorker_DoWork(Object sender, DoWorkEventArgs e) in c:\P\ZetaResourceEditor\Source\Main\UI\ExportImportExcel\ExcelExportWizardForm.cs:line 749
   at System.ComponentModel.BackgroundWorker.OnDoWork(DoWorkEventArgs e)
   at System.ComponentModel.BackgroundWorker.WorkerThreadStart(Object argument)

DuplicateNameException
{
	Message="A DataTable named \\'00000000-0000-0000-0000-0000...\\' already belongs to this DataSet.",
	Data=
	[
	]
,
	InnerException=null,
	TargetSite=RuntimeMethodInfo: RegisterName,
	StackTrace="   at System.Data.DataTableCollection.RegisterName(String name, String tbNamespace)\\r\\n   at System.Data.DataTable.set_TableName(String value)\\r\\n   at ZetaResourceEditor.RuntimeBusinessLogic.ExportImportExcel.Export.ExcelExportController.doProcess(PreparedInformation preparedInformation, BackgroundWorker bw, Int32 fileGroupsLength, ICollection`1 savedFiles) in c:\\P\\ZetaResourceEditor\\Source\\RuntimeBusinessLogic\\ExportImportExcel\\Export\\ExcelExportController.cs:line 451\\r\\n   at ZetaResourceEditor.RuntimeBusinessLogic.ExportImportExcel.Export.ExcelExportController.Process(BackgroundWorker bw) in c:\\P\\ZetaResourceEditor\\Source\\RuntimeBusinessLogic\\ExportImportExcel\\Export\\ExcelExportController.cs:line 304\\r\\n   at ZetaResourceEditor.UI.ExportImportExcel.ExcelExportWizardForm.progressBackgroundWorker_DoWork(Object sender, DoWorkEventArgs e) in c:\\P\\ZetaResourceEditor\\Source\\Main\\UI\\ExportImportExcel\\ExcelExportWizardForm.cs:line 749\\r\\n   at System.ComponentModel.BackgroundWorker.OnDoWork(DoWorkEventArgs e)\\r\\n   at System.ComponentModel.BackgroundWorker.WorkerThreadStart(Object argument)",
	HelpLink=null,
	Source="System.Data",
	HResult=-2146232030
}```

**Comment by UweKeim** [comment]
Thanks, I'll take a lool.

**Comment by UweKeim** [comment]
I've tried to reproduce this error but unfortunately until now I don't manage to create this error.

Could you help me by:

- Show me step-by-step screenshots of the export wizard and its values -and/or-
- send me an example project to generate the error

Would be very helpful.

**Comment by ArneD** [comment]
It worked after restarting the program.

However, I could reproduce the error with the following steps:
 * Create new project
 * Add existing file group to project
 * Add atleast 2 existing file groups
 * Export to Excel
 * Select all the files and all the languages
 * Select 'Export each file group into seperate work sheet'
 * With the following options:
![image](https://cloud.githubusercontent.com/assets/8726691/20557133/6d7a3534-b16a-11e6-9459-047712637b66.png)

Hope this helps!

**Comment by UweKeim** [comment]
Seems that you have unsaved items. The export wizards seems to not check that.

I've just published a new version (**2.3.0.5**), that _could_ fix this, although I still don't fully understand what could lead to the error condition.

Maybe you could give [the new setup](http://zeta.li/zre-download-setup) a try.

**Comment by ArneD** [comment]
Seems to be working now!
